bottlecap/src/lifecycle/flush_control.rs

```diff
@@ -37,7 +37,7 @@ impl FlushControl {
         let now = match time::SystemTime::now().duration_since(time::UNIX_EPOCH) {
             Ok(now) => now.as_secs(),
             Err(e) => {
-                debug!("failed to get current time: {:?}", e);
+                debug!("Failed to get current time: {:?}", e);
```

Comment:
I think this should be very unlikely, should be warn/err?

Reply:
Yeah, maybe an error

Reply:
Actually, this is removed during my racing flushing PR, won't fix